packages/xod-arduino/src/messages.js

```diff
@@ -7,4 +7,28 @@ export default {
     solution:
       'Try to express the node as a XOD patch node composed of smaller C++ nodes or group the outputs into a new custom type.',
   }),
+  GLOBAL_LITERAL_VALUE_MISSING: ({ key }) => {
+    const messages = {
+      DEFAULT: {
+        title: `Literal "${key}" needs a value`,
+        note: `The program uses the \`=${key}\` literal, but XOD IDE does not know the value for this literal.`,
+        solution:
+          'Report the bug on the forum and attach the xodball if possible',
```

Comment:
I think if one entered `=FOO` to a field, that’s not the reason to report a bug on the forum. Do you mean: “Check the literal spelling” ?